2D_NRSupport_FluxModel.py

```diff
@@ -207,15 +222,15 @@ def dSVdt_func(t,SV): # System of ODEs to solve
     def animate(i):
         ax.collections = []
         SV_plt = np.reshape(SV[i,plt_species_ind::Nspecies],(Ny,Nx))
-        cf = ax.contourf(x_plt,y_plt,SV_plt,
-                         levels=np.linspace(np.min(SV_cb),np.max(SV_cb),Nc))
+        cf = ax.imshow(SV_plt,extent=[0,1e6*(rad+max_space),t_glass,0])
+
         cax.cla()
         fig.colorbar(cf, cax=cax)
         ax.set_title('Time (s): ' + str(round(sol_t[i])))
         
     anim = FuncAnimation(fig, animate, interval=10, frames=len(sol_t)-1)
     
-    anim.save('animation.html')
+    anim.save(movie_file_name)
 

```

Comment:
Would be good to add a saving routine that does the following:

-Creates a folder to hold all output from the run.
-Saves the current version of the all model files and the cti file into this folder
-Saves the full simulation solution (`solution.y`) into this folder
-Saves all figures and the movie into the folder.